### DISCUSSION TITLE
Orama Cloud: Edit a document in an index?

### DISCUSSION THREAD

**Discussion by osseonews** [oramasearch/orama#676] [Q&A]
After you upload an index, how do you edit documents? From what I can tell on Orama cloud, you can't do anything with the Edit features, except to edit the searchable properties. But, let's say we need to edit a specific document in the index, how do we do that? Do we need to upload the entire JSON file again, and reindex everything, if just need to make one small change?

**Comment by micheleriva** [comment]
If you're using a JSON file as a datasource, yes. For production-grade deployments, I'd recommend using the WebHook APIs: https://docs.oramasearch.com/cloud/data-sources/custom-integrations/webhooks#updating-removing-inserting-elements-in-a-live-index
